app/assets/stylesheets/components/indicators.scss

```diff
@@ -56,11 +56,7 @@
     var(--indicator-critical-color)
   );
 
-  // Circle, no text
-  &--bullet {
-    width: var(--su-3);
-    height: var(--su-3);
-    padding: 0;
-    border-radius: 100%;
+  &:empty {
```

Comment:
This prevents displaying an empty indicator which used to be handled somewhere else as a "one off"